README.md

```diff
@@ -1 +1,45 @@
-# Learning locker docker
+# Learning locker docker images
+
+This repo contains the Dockerfile to build images for [learning locker](https://github.com/LearningLocker/learninglocker), a LRS to store and analyze xAPI statements.
+
+We follow the learning locker's tag, so an image will have the same tag you can found in learning locker repository.
+
+## Building a learning locker image
+
+To build a learning locker image, you have to pick the right tag to use, set it in the `VERSION` environment variable and then use the `build` task in our Makefile:
+
+```bash
+$ VERSION="v2.6.2" make build
+```
+
+## Use learning locker image
+
+You have to configure your learning locker image using environment variable. You can found all the variable available in the [documentation](http://docs.learninglocker.net/guides-configuring/#learning-locker-application).
+
+You also have to use the [xapi-service](https://github.com/LearningLocker/xapi-service) provided by learning locker. They also provide a docker image you can use.
+
+There is no entrypoint nor command configured in this image. You will have to declare them when you run your container. Here are the command to use:
+
+Running the UI application:
+
+```
+node ui/dist/server
```

Comment:
I usually like to add a shell prompt so that we understand that those are shell commands. WDYT?

```suggestion
$ node ui/dist/server
```

Reply:
Yes but you don't execute it in a shell. You will add it in a `command` instruction, this is why I didn't put `$` at the beginning